services/QuillLMS/client/app/bundles/Shared/components/shared/list.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import Checkbox from './checkBox';
+
+type StyleType = 'primary' | 'secondary' | 'checkbox'
+interface ListProps {
+  items: any,
+  style: StyleType
+}
+
+const PRIMARY = 'primary'
+const SECONDARY = 'secondary'
+const CHECKBOX = 'checkbox'
+
+export const List = ({ items, style }: ListProps) => {
+  if(style === PRIMARY) {
+    return(
+      <ul className="list quill-list single-line">
+        {items.map(({ label, onClick }) => (
+          <li>
+            <button className="interactive-wrapper" onClick={onClick}>{label}</button>
```

Comment:
I think this button probably needs a `type` to pass the linter, right? I also wonder if we want to give it a `focus-on-light` class.

Reply:
I think that's a reasonable approach-- I just assumed by the focus and active states that these would always be interactive elements. I'll add logic to check for a click handler and if none is present, just render a regular `li` element